controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/AutoScaleTask.java

```diff
@@ -194,13 +195,14 @@ public AutoScaleTask(final StreamMetadataTasks streamMetadataTasks,
      * @return CompletableFuture
      */
     private CompletableFuture<Void> postScaleRequest(final AutoScaleEvent request, final ArrayList<Long> segments,
-                                                     final ArrayList<AbstractMap.SimpleEntry<Double, Double>> newRanges) {
+                                                     final ArrayList<AbstractMap.SimpleEntry<Double, Double>> newRanges,
+                                                     final long timestamp) {
         ScaleOpEvent event = new ScaleOpEvent(request.getScope(),
                 request.getStream(),
                 segments,
                 newRanges,
                 false,
-                System.currentTimeMillis());
```

Comment:
please revert this change. The timestamp in request corresponds to when hot or cold state of a segment was reported by segment store. 

the timestamp of when a scale is requested can be different and hence we use System.currentTime as the time when we post the scale request.

Reply:
Change reverted. Now `ScaleOpEvent` contains a `requestId` field.

Reply:
Change reverted. Now `ScaleOpEvent` contains a `requestId` field.